docs/src/css/style.module.css

```diff
@@ -0,0 +1,608 @@
+@import url("https://fonts.googleapis.com/css2?family=Chivo:ital,wght@0,100..900;1,100..900&display=swap");
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  width: 100%;
+  margin-inline: auto;
+  padding-inline: 12px;
+}
+
+@media (min-width: 576px) {
+  .container {
+    max-width: 540px;
+  }
+}
+/* Small */
+@media (min-width: 768px) {
+  .container {
+    max-width: 720px;
+  }
+}
+/* Medium */
+@media (min-width: 992px) {
+  .container {
+    max-width: 960px;
+  }
+}
+/* Large */
+@media (min-width: 1200px) {
+  .container {
+    max-width: 1140px;
+  }
+}
+/* Extra large */
+@media (min-width: 1400px) {
+  .container {
+    max-width: 1254px;
+  }
+}
+/* Fluid container */
+.container-fluid {
+  width: 100%;
+  padding-right: var(--bs-gutter-x, 0.75rem);
+  padding-left: var(--bs-gutter-x, 0.75rem);
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.cmnButton {
+  display: inline-block;
+  padding: 4px 35px 6px;
+  color: #000000 !important;
+  background-color: #eacc8b;
+  border: 1px solid #eacc8b;
+  font-size: 20px;
+  text-decoration: none;
+  transition: all 0.3s linear;
+}
+.cmnButton:hover {
+  color: #eacc8b !important;
+  background-color: transparent;
+}
+.cmnButton.outline {
+  color: #eacc8b !important;
+  background-color: transparent;
+}
+.cmnButton.outline:hover {
+  color: #000000 !important;
+  background-color: #eacc8b;
+  border: 1px solid #eacc8b;
+}
+
+.munheader {
```

Comment:
In an effort to fix the styling, it looks like Munna added a prefix to a bunch of these css classes: [`011ae98` (#278)](https://github.com/groundlight/python-sdk/pull/278/commits/011ae98a83447cf1f8f2577c02a0234007de72cc)

Can we clean this up by doing one of the following:
1. Using a better prefix that isn't a name/alias? Maybe like "landingpage" if these all pertain to the landing page.
2. Wrapping in some sort of namespace? Is this a thing in css?

Reply:
Yeah sounds good!